### DISCUSSION TITLE
local lspconfig = require('lspconfig') lspconfig.gdscript.setup {} 应该放哪啊？

### DISCUSSION THREAD

**Discussion by imac8t** [ayamir/nvimdots#1358] [Q&A] [answered]
local lspconfig = require('lspconfig') lspconfig.gdscript.setup {} 应该放哪啊？

**Comment by charliie-dev** [comment, marked as answer]
https://github.com/ayamir/nvimdots/wiki/Usage#modify-lsps-linters-and-formatters
